app/src/main/java/com/example/filemanager/AllFilesFragment.kt

```diff
@@ -50,26 +40,18 @@ class AllFilesFragment : Fragment(), ItemClickListener {
                 RecyclerView.VERTICAL
             )
         )
+        val view = binding.root
+        return view
```

Comment:
the view variable seems to be redundant, it is not used. Can simply return the `binding.root`